scripts/install_terraform.sh

```diff
@@ -1,13 +1,7 @@
 #!/bin/bash
 
-FILE=terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+# install tfswitch
+curl -sSL https://raw.githubusercontent.com/warrensbox/terraform-switcher/release/install.sh | bash
```

Comment:
A bit out of scope, but after CodeCov incident we might not want to execute files from the scary internet without some reassurance that we downloaded what we wanted.

Reply:
this script actually verifies the checksum: https://github.com/warrensbox/terraform-switcher/blob/9023d0171f4124f310366a50bb21c858c3cafd7f/install.sh#L312 if we want to do this explicitly we'd need to save this script locally and run it form there 🤔  or how would you do it?

Reply:
I guess if we want to be on the save side we should save the checksum of the script or run it locally.

Reply:
We did the same so far here too (grabbing things from the internet), so that's why I'm saying it's a bit out of scope now and we can iterate it on later too.

Reply:
did this, pls check out the latest commit